apps/super-app/src/services/ServiceSupportDetails.ts

```diff
@@ -86,3 +85,339 @@ const PRO_BONO_SG_DETAIL: Record<LanguageOption, SupportDetail> = {
+    carouselSection: [],
+    blurbFacebookLinks: {
+      title: '',
+      imageSrc: '/images/facebook-icon.svg',
+      links: [
+        {
+          href: 'https://www.facebook.com/sgadeo',
+          text: 'Facebook page',
+        },
+      ],
+    },
+    ctaButtonText: 'Message us',
+    ctaLink: 'tel:+65%206304%203482',
+    ctaIcon: '',
+  },
+};
+
+const SGACCIDENTHELPCENTRE_DETAIL: Record<
+  LanguageOption,
+  SupportDetail
+> = {
+  en: {
+    logo: 'https://static.wixstatic.com/media/862048_3725dd21f0d64ef492b3e5665c257ab5~mv2.jpg/v1/crop/x_46,y_63,w_725,h_433/fill/w_219,h_131,al_c,q_80,usm_0.66_1.00_0.01,enc_auto/IMG-20180416-WA0009%20(1).jpg',
+    name: 'SG Accident Help Centre',
+    website: 'https://www.sgaccident.org.sg/about',
+    blurbIntro:
+      'We are here for the injured and needy. We want to care for injured migrant workers in the community and teach employed workers on workplace safety and injury prevention',
+    carouselSection: [],
+    blurbFacebookLinks: {
+      title: '',
+      imageSrc: '/images/facebook-icon.svg',
+      links: [
+        {
+          href: 'https://www.facebook.com/pages/SG%20Accident%20Help%20Centre/335942880619911/',
+          text: 'Facebook page',
+        },
+      ],
+    },
+    ctaButtonText: 'Call us',
+    ctaLink: 'tel:+65%206291%200751',
+    ctaIcon: '',
+  },
+};
+
+const MWC_DETAIL: Record<LanguageOption, SupportDetail> = {
+  en: {
+    logo: '/images/mwc-logo.jpeg',
+    name: "Migrant Workers' Centre",
+    website: 'https://mwc.org.sg/wps/portal/mwc/home/Aboutus/whatwedo',
+    blurbIntro:
+      'We can advise you about unfair employment practices and provide shelter and food for those in need. We also provide membership to protect you against critical illness, death, and accidents as well as meaningful benefits',
+    carouselSection: [],
+    blurbFacebookLinks: {
+      title: 'Explore our services',
+      imageSrc: '/images/link-icon.png',
+      links: [
+        {
+          href: 'https://mwc.org.sg/wps/portal/mwc/home/services/associatemembership/',
+          text: 'MWC Associate Membership',
+        },
+        {
+          href: 'https://mwc.org.sg/wps/portal/mwc/home/services/freelegalclinic/',
+          text: 'Free Legal Clinic',
+        },
+      ],
+    },
+    ctaButtonText: 'Call us',
+    ctaLink: 'tel:+65%206536%202692',
+    ctaIcon: '',
+  },
+};
+
+const EMPTY_DETAIL: Record<LanguageOption, SupportDetail> = {
+  en: {
+    logo: '',
+    name: '',
+    website: '',
+    blurbIntro: '',
+    carouselSection: [],
+    blurbFacebookLinks: {
+      title: '',
+      imageSrc: '',
+      links: [
+        {
+          href: '',
+          text: '',
+        },
+        {
+          href: '',
+          text: '',
+        },
+      ],
+    },
+    ctaButtonText: '',
+    ctaLink: '',
+    ctaIcon: '',
+  },
+};
+
+export function getPartnerDetail(
+  partner: string | undefined,
```

Comment:
nit, really minor thing: for more type-safety, maybe this could use `ServicePath` instead of `string`?